src/main/webapp/WEB-INF/view/login.jsp

```diff
@@ -39,11 +39,15 @@
         <h2 style="color:red"><%= request.getAttribute("error") %></h2>
     <% } %>
 
-    <form action="/login" method="POST">
-      <label for="username">Username: </label>
-      <input type="text" name="username" id="username">
-      <button type="submit">Login</button>
-    </form>
-  </div>
+   <form action="/login" method="POST">
+     <label for="username">Username: </label>
+     <input type="text" name="username" id="username">
+     <br/>
```

Comment:
I am guessing that you're using br/ as a way to space things out a little bit. While that probably works here, and I don't care about it that much, the usual way would be to use CSS to get padding correct.  (No action required, just an observation.)